apps/cbportal/base/CBGrid/Images/Image.wiki

```diff
@@ -4,6 +4,9 @@
 {% if id %}
 h3. Image: ${name}
 
+{% if status != "DESTROYED" %}
+{% if status != "DELETED" %}
```

Comment:
two ifs and 2 endifs  you can just do an `and` or `in` right?

Reply:
No this is needed.